exporter/clickhouseexporter/internal/gauge_metrics.go

```diff
@@ -70,6 +70,7 @@ SETTINGS index_granularity=8192, ttl_only_drop_parts = 1;
     MetricDescription,
     MetricUnit,
     Attributes,
+    StartTimeUnix,
```

Comment:
How is the upgrade experience going to be with such a change? The table will not be recreated automatically, and the new version of the exporter won't work until the table is recreated, right?

Reply:
Correct. What's the guarantee here in general? Do we follow semver? We could mention that breaking change in the readme.

Reply:
The table create SQL already has column `StartTimeUnix`, @dmitryax your worries won't happen.

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18288/files#diff-f7bdb65e4ce3de30d08997f907af2cc3ce6faf3ae319d6473d72d003936e4617L43 

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18288/files#diff-e10408b52409e5277dc13b218531e360d5988267347bb33b58fb28ec32b12b55L43

Reply:
You're correct.

Reply:
> The table create SQL already has column StartTimeUnix, @dmitryax your worries won't happen.

You pointed to this PR updating the create table query. What do you mean by "your worries won't happen"?

Reply:
> What's the guarantee here in general? 

I believe it's ok given that the component is in alpha stability state. 

> We could mention that breaking change in the release notes.

I believe we should mark it as a breaking change in the changelog entry and say that the upgrade requires removing up upgrading the old table

Reply:
Sounds good.

Reply:
> You pointed to this PR updating the create table query. 

This PR not update the create table query but only change the insert table query. 

> What do you mean by "your worries won't happen"?

It means the new version of the exporter will work fine without the table is recreated.

Reply:
Oh, Ok, I missed that. Sorry